### PR TITLE
Add helmet to improve default security

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

**Comment by digitalsadhu** [comment]
@pnw

File: src/classes/application.js [hunk 1]
```diff
@@ -160,6 +161,8 @@ module.exports = class Application extends Base {
     const app = express()
     _app.set(this, app)
 
+    app.use(helmet())
```
Comment (pnw) [inline src/classes/application.js]: Hmm is it presumptuous to assume everyone is going to want helmet? Should it be something you can turn off in the config? Id say that ceding control to the end user is good - helmet is on by default, but they can turn it off with `ENV.helmet = false` or `ENV.middleware.helmet = false` or something.

**Comment by digitalsadhu** [comment]
It's considered a best practice as listed on the express website. We could
provide a way to turn it off in congress I guess. That's probably a good
idea.

**Comment by digitalsadhu** [comment]
Did this work on master

**Pull request closed without merging**